src/components/contact-footer.astro

```diff
@@ -0,0 +1,141 @@
+<section class="contact-footer">
+  <div class="container">
+    <section class="content">
+      <h2>
+        Vi vil gjerne høre fra deg!
+      </h2>
+      <p>
+        Uansett om du har en idé som vil forandre verden, eller om det er en liten sak som irriterer deg i hverdagen, så er vi like ivrige.
+      </p>
+      <p>
+        Ta kontakt med oss for en hyggelig kaffeprat!
+      </p>
+      <form action="">
+        <div class="identity">
+          <input placeholder="Fullt navn" type="text" name="name" id="name" required />
+          <input placeholder="E-postadresse" type="email" name="email" id="email" required />
+        </div>
+        <div class="issue">
+          <input placeholder="Hva dreier deg seg om?" type="text" name="text" id="text" required />
+        </div>
+        <div class="buttons">
+          <div class="accept">
+            <input type="checkbox" name="personvern" id="" value="I AM HERE ok" required>  
+            <p>Jeg bekrefter at jeg har lest og akseptert vår <a href="/personvern" target="_blank">personvern­erklæring</a>.</p>
```

Comment:
Do we need the gdpr in this context? The end result we want here is just to submit the data to something like fresh / email with "I want to talk with you." which they explicitly submit for that purpose. We won't be using that data anywhere else

Reply:
Wall of text warning. tl;dr a GDPR checkbox allows us to do more with the data (email etc.) than just fulfill their request.

GDPR applies whenever you're collecting personal information. Basically anything is personal information, the definition is quite broad since since it doesn't take much to identify something. Here that would be collecting someone's email address.

The thorny thing then is what purpose do we process it for. Here you can use the ones listed in [article 6](https://gdpr.eu/article-6-how-to-process-personal-data-legally/) 1) a thru f (it may be more than one).

If we *only* use it for the purpose of contacting the sender regarding their message then that is allowed under art. 6 1)b since we need to use it to fulfill our contract to contact them. We do not need to ask for consent. 
However when that is finished we are not allowed to use it for any other purpose (we only used 1)b as a justification to process it) and should delete it. So if we want to store emails for whatever reason we need to get their permission.

I'm mixing the terms collecting and processing personal data, but they're basically the same. You are processing personal data whenever you have a chunk of them that are organized in any way. So your email account is one example.